source/faq.txt

```diff
@@ -24,13 +24,25 @@ What Is the Difference Between "connectTimeoutMS", "socketTimeoutMS" and "maxTim
      - Description
    * - connectTimeoutMS
      - 30000
-     - ``connectTimeoutMS`` is a `connection-setting <https://mongodb.github.io/node-mongodb-native/3.5/reference/connecting/connection-settings/>`_ option that sets the time in milliseconds to establish a connection to the MongoDB server before timing out.
+     - ``connectTimeoutMS`` is a :doc:`connection option
+       </fundamentals/connection#connection-options>` that sets the time in
+       milliseconds for an individual connection from your connection pool to
+       establish a TCP connection to the MongoDB server before timing out.
+       Use the ``serverSelectionTimeoutMS`` option instead to fail when
+       :node-api:``MongoClient.connect <MongoClient.html#.connect>` is unable
+       to establish a connection with the MongoDB server.
    * - socketTimeoutMS
      - 360000
-     - ``socketTimeoutMS`` is a ``connection-settings`` option that sets the number of milliseconds a socket stays inactive after the driver has successfully connected before closing the connection.
+     - ``socketTimeoutMS`` is a :doc:`connection option
```

Comment:
a few things here:
- `socketTimeoutMS` is a connection option as well as an option supported on individual operations
- I think the option is better described this way: "`socketTimeoutMS` is the amount of time the driver will wait for an inactive socket before closing it. The default is to never time out the socket. This only applies to sockets which have already been connected"

Reply:
Is the default value of "360000" incorrect since in your second sentence, you mention that the default is to never time out? Or does the default apply only when it is a connection option and not for individual operations?

Reply:
Hm, you're right that for some reason the legacy options apply a 360000 default, but the idea is that this is effectively "forever" and is actually specified as such. Let's say its no timeout for v4, we'll fix that by then